packages/ui/lib/components/Form/FieldLayout.tsx

```diff
@@ -59,9 +59,9 @@ export function FieldLayout(props: Props) {
 
     if (description) {
       return (
-        <p id={label} className={descriptionClass}>
```

Comment:
This was causing a warning because sometimes `description` is a JSX object with `<p>` in it and you can nest blocks!